src/main/java/com/gw/web/GeoweaverController.java

```diff
@@ -57,65 +57,71 @@
  * Controller for SSH related activities, including all the handlers for Geoweaver.
  * This controller handles various web requests related to Geoweaver's functionality.
  */
-@Controller 
+@Controller
 @RequestMapping(value="/web")
 public class GeoweaverController {
 
     // Logger for logging controller-related activities.
     Logger logger = LoggerFactory.getLogger(getClass());
-    
+
     // Autowired tools and services required for various operations.
     @Autowired
     ProcessTool pt;
-    
+
     @Autowired
     WorkflowTool wt;
 
     @Autowired
     WorkflowRepository workflowrepository;
-    
+
     @Autowired
     HostTool ht;
-    
+
     @Autowired
     BaseTool bt;
-    
+
     @Autowired
     GWSearchTool st;
-    
+
     @Autowired
-    FileTool ft;
-    
+	FileTool ft;
```

Comment:
There must be some problem with your IDE. Why the space before `FileTool` is weird. @gokulprathin8